common/lib/xmodule/xmodule/course_module.py

```diff
@@ -303,6 +303,12 @@ class CourseFields(object):
         help=_("Enter the unique identifier for your course's video files provided by edX."),
         scope=Scope.settings
     )
+    facebook_url = String(
```

Comment:
@gwprice I asked this question to Lou in the mobile chat room earlier this week.  Here's his response:

```
@LouWang For the new Studio Advanced Setting for the URL to the Official Course FB page, does it make sense to keep it specific to FB or generalize it as Social Group URL?	
10:54 AM Lou Wang i think it should be specific, since the app UI exposes it as a FB link
```

For the API, however, I have generalized it by including FB as one of a list of social options.